ONSdigital/dp-api-tests
# Feature/dataset api update error messages

### What

Sync tests to changes made in dataset api to return lower cased error messages in http responses

### How to review

Make sure tests pass, by running the following:
1) Make sure mongodb, neo4j, kafka and zookeeper are running
2) Run `dp-auth-api` with: `make debug`
3) Checkout `feature/audit-instance-package` on `dp-dataset-api` and run the following command: `make acceptance-publishing`
4) Make sure healthcheck has occurred on your running `dp-dataset-api` instance
5) In `dp-api-tests` run the following command: `go test publishing/datasetAPI/.`
6) Stop `dp-dataset-api` instance and run it again in web mode: `make acceptance-web`
7) Wait for healthcheck to occur before running the following in `dp-api-tests`:
`go test web/datasetAPI/.`

Both test runs should pass

See dataset API pull request [here](https://github.com/ONSdigital/dp-dataset-api/pull/161)

### Who can review

Anyone
